libraries/AC_Planck/AC_Planck.h

```diff
@@ -66,6 +66,8 @@ class AC_Planck {
   //Get a position, velocity, yaw command
   bool get_posvel_cmd(Location &loc, Vector3f &vel_cms, float &yaw_cd, bool &is_yaw_rate);
 
+  uint32_t mux_rates(const float rate_up, const float rate_down){return (uint16_t(rate_up) << 16) | uint16_t(rate_down);};
```

Comment:
Have you verified this? I worry that due precedence of operations uint16_t(rate_up) << 16 = 0. You may want to use uint32_t casts instead.

Reply:
Hasn't been verified yet, but I'll make the change either way